### PR TITLE
add missing packages when using cinder backend

### PR DESCRIPTION
**Pull request by guilhermesteinmuller**
to address https://github.com/vexxhost/atmosphere/issues/36

### CODE REVIEW

File: ci/openstack-image-build-env [hunk 1]
```diff
@@ -10,7 +10,7 @@ PROFILES = {
     "horizon": "apache",
     "monasca-api": "apache influxdb",
     "ironic": "ipxe ipmi qemu tftp",
-    "glance": "ceph",
+    "glance": "ceph os_brick python-cinderclient",
```
Comment (mnaser) [inline ci/openstack-image-build-env:13]: This is `PROFILES`, you would need to add it into `PIP_PACKAGES`.

Let me work on fixing it.

**Review by mnaser — CHANGES_REQUESTED** [review]

File: ci/openstack-image-build-env (hunk 1 above)
Comment (guilhermesteinmuller) [reply ci/openstack-image-build-env]: @mnaser yes my bad, sorry. I pushed a fix

**Comment by guilhermesteinmuller** [comment]
@mnaser kind ping here :) sorry for the mess

**Comment by mnaser** [comment]
FYI, I am closing this because I handled it in https://github.com/vexxhost/atmosphere/pull/38.

**Pull request closed without merging**